### ISSUE TITLE
Image Gallery Controls Not Rendering

### ISSUE THREAD

**Issue by pt2302** [mitodl/ocw-hugo-themes#1057]
### Steps to Reproduce

The image gallery controls (previous/next buttons) are not rendering properly. This may be a Unicode issue. For an example, see [this image gallery](https://ocw.mit.edu/courses/2-672-project-laboratory-spring-2009/pages/labs/lab-3-heat-exchanger-tube-vibrations/#nanogallery/042bd21e-b090-206a-b8f9-deead93d605d_nanogallery2/0/2).

**Comment by gumaerc** [comment]
Seems to be an issue with a missing font:

`downloadable font: download failed (font-family: "ngy2_icon_font" style:normal weight:400 stretch:100 src index:0): status=2147746065 source: https://ocw.mit.edu/static/css/fonts/[fullhash].woff2`

**Comment by HussainTaj-arbisoft** [comment]
I've discovered that the nanogallery font files are being removed in a webpack rule.

```ts
      {
        test: /nanogallery2/,
        use:  "imports-loader?module.exports=>undefined&exports=>undefined"
      }
```

I've experimented with a few options and setting the rule's regex to ```/nanogallery2(?!.*\.(woff|woff2)$)/``` seems to do the trick. I'm inclined to exclude only the specific type of fonts this fix needs. This is to avoid reversing any effects of the resolution of https://github.com/mitodl/ocw-hugo-themes/issues/1033.

Additionally, the rule that loads the app's fonts places them under `static_shared/fonts`. The nanogallery module looks for the fonts under `static_shared/css/fonts` (likely due to a relative import).

Adding a rule specific to nanogallery fonts should work.
